server/src/test/java/io/crate/protocols/http/HeadersTest.java

```diff
@@ -35,42 +35,52 @@ public class HeadersTest {
 
     @Test
     public void testIsBrowser() {
-        assertThat(isBrowser(null), is(false));
-        assertThat(isBrowser("some header"), is(false));
-        assertThat(isBrowser("Mozilla/5.0 (X11; Linux x86_64)"), is(true));
+        assertThat(isBrowser(null)).isFalse();
+        assertThat(isBrowser("some header")).isFalse();
+        assertThat(isBrowser("Mozilla/5.0 (X11; Linux x86_64)")).isTrue();
     }
 
     @Test
     public void testIsAcceptJson() {
-        assertThat(isAcceptJson(null), is(false));
-        assertThat(isAcceptJson("text/html"), is(false));
-        assertThat(isAcceptJson("application/json"), is(true));
+        assertThat(isAcceptJson(null)).isFalse();
+        assertThat(isAcceptJson("text/html")).isFalse();
+        assertThat(isAcceptJson("application/json")).isTrue();
     }
 
     @Test
     public void testExtractUsernamePasswordFromHttpBasicAuthHeader() {
-        Credentials creds = extractCredentialsFromHttpBasicAuthHeader("");
-        assertThat(creds.username(), is(""));
-        assertThat(creds.password().toString(), is(""));
+        Credentials creds = extractCredentialsFromHttpAuthHeader("");
+        assertThat(creds.username()).isNull();
+        assertThat(creds.password()).isNull();
 
-        creds = extractCredentialsFromHttpBasicAuthHeader(null);
-        assertThat(creds.username(), is(""));
-        assertThat(creds.password().toString(), is(""));
+        creds = extractCredentialsFromHttpAuthHeader(null);
+        assertThat(creds.username()).isNull();
+        assertThat(creds.password()).isNull();
 
-        creds = extractCredentialsFromHttpBasicAuthHeader("Basic QXJ0aHVyOkV4Y2FsaWJ1cg==");
-        assertThat(creds.username(), is("Arthur"));
-        assertThat(creds.password().toString(), is("Excalibur"));
+        creds = extractCredentialsFromHttpAuthHeader("Basic QXJ0aHVyOkV4Y2FsaWJ1cg==");
+        assertThat(creds.username()).isEqualTo("Arthur");
+        assertThat(creds.password().toString()).isEqualTo("Excalibur");
```

Comment:
Super minor, no need to change, just FYI:
```suggestion
        assertThat(creds.password()).hasToString("Excalibur");
```